src/app.jsx

```diff
@@ -101,7 +106,37 @@ export const App = () => {
                 .finally(() => setLoadingResources(false));
     }, []);
 
-    if ((superuser.allowed && systemSocketInactive) || loadingResources) {
+    useEffect(() => {
+        async function checkVirtualization() {
+            const hardwareVirtCheck = await cockpit.script("virt-host-validate | grep 'Checking for hardware virtualization'");
+
+            setVirtualizationEnabled(hardwareVirtCheck.includes('PASS'));
+        }
+
+        checkVirtualization();
+    }, []);
+
+    if (!virtualizationEnabled && !emptyStateIgnored) {
+        return (
+            <EmptyState>
+                <EmptyStateIcon icon={WrenchIcon} />
+                <Title headingLevel="h4" size="lg">
+                    {_("Virtualization support is not available")}
```

Comment:
'Hardware virtualization is disabled' is more correct. is not available sounds like you can't undo it. THis was the suggestion from Martin and i like it.

Reply:
Yes, it looks better. Changed it.

Reply:
> Hardware virtualization is disabled

Good call. I like that too.